docs/common/craft-parts/reference/plugins/ant_plugin.rst

```diff
@@ -3,6 +3,7 @@
 Ant plugin
 ==========
 
+.. _craft_parts_ant_plugin_description_begin:
```

Comment:
should we scope this further and only surround the "After a successful build..." plus the steps? This initial paragraph is still true for all *crafts right?

Maybe `_craft_parts_ant_plugin_post_build_begin/end`